egs/chime4/s5_1ch/run.sh

```diff
@@ -7,11 +7,22 @@
 #                Mitsubishi Electric Research Labs (Shinji Watanabe)
 #  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
 
+# Added advanced baseline
+# Szu-Jui Chen 10/19/2017
+
 . ./path.sh
 . ./cmd.sh
+#####Baseline settings#####
+# Usage: 
+# For using original baseline, set baseline to chime4_official. In this case,
+# you don't need to run flatstart since we already provided the trained model in 
+# tools/ASR_models.
+# For advanced baseline, please run flatstart first to get the models.
+# By default, we assume you already got the models.
 
 # Config:
 stage=0 # resume training with --stage=N
+baseline={"chime4_official","advanced"}
```

Comment:
I don't know what you are trying to do here. I think maybe what you mean is something like this:


```
baseline=advanced
# By default this runs the 'advanced' baseline.  Uncomment the following to run
# the 'chime4_official' (i.e. less advanced) baseline.
# baseline=chime4_official
#

```